### ISSUE TITLE
Can not reproduce the SPIN training results.

### ISSUE THREAD

**Issue by ykk648** [open-mmlab/mmhuman3d#145]
Ref official SPIN training [log](https://github.com/open-mmlab/mmhuman3d/tree/main/configs/spin/), the loss is about 0.3-0.5, my training loss can only reached 0.7:
[20220330_073642.log](https://github.com/open-mmlab/mmhuman3d/files/8385298/20220330_073642.log)
 
Evaluated on 3DPW, got PA-MPJPE 61, MPJPE 101

check SPIN fits from [spin](https://github.com/open-mmlab/mmhuman3d/tree/main/configs/spin/)
check pretrain model, using SPIN/HMR pretrain got same result, the loss is unnormal since first step as the log shows.

The only difference may be `spin_h36m_train.npz` , I generate it from mosh data, size is 341.7m on ubuntu, compared with 169.3m that mmhuman3d provides.

Can you tell me why this happend and how to reproduce the SPIN training results?

**Comment by caizhongang** [comment]
Hi @ykk648 , we also provide the preprocessed .npz files in [Data Preparation](https://github.com/open-mmlab/mmhuman3d/blob/main/docs/preprocess_dataset.md).
Specifically, you can download [spin_h36m_train.npz](https://openmmlab-share.oss-cn-hangzhou.aliyuncs.com/mmhuman3d/datasets/spin_h36m_train.npz?versionId=CAEQHhiBgID657fT6xciIDM4ZDc5YjIwZjQwOTQ4ODRhOWZhMWQwZmVhNWZkZTQ3). Please let us know if this addresses your issue.

**Comment by ykk648** [comment]
I check again, even the *.npz is same as offered, the loss of first 50 step is still unnormal.
I take all *.npz but `spin_mpi_inf_3dhp_train.npz` generated by myself, cause the frame name is `frame_xxx.png` .

more warning logs may be helpful
[warning_log.txt](https://github.com/open-mmlab/mmhuman3d/files/8386338/warning_log.txt)

---

What I have tried:
1.only change image_path in mmhuman3d version `spin_mpi_inf_3dhp_train.npz`, same result
2.change `partition` in config, when set mpi_inf_3dhp ratio to 0. , the first step loss seems normal (using SPIN pretrain)
3.noticed the recent commit #140,  have tried regenerate `mpi_inf_3dhp_train.npz` and `spin_mpi_inf_3dhp_train.npz`, same result

**Comment by ykk648** [comment]
update:

using eft dataset to replace spin*.npz， 3DPW PA-MPJPE reached 59.03

**Comment by Vegetebird** [comment]
@ykk648 Hi~Could you share me the mosh data? My wechat is: Vegetaberd or email is: [781256005@qq.com]

**Comment by ykk648** [comment]
@Vegetebird try https://github.com/Jeff-sjtu/HybrIK/issues/5#issuecomment-820378519

**Comment by flyyyyer** [comment]
@ykk648 @caizhongang I use the provide the preprocessed .npz files in [Data Preparation](https://github.com/open-mmlab/mmhuman3d/blob/main/docs/preprocess_dataset.md] or spin_h36m_train.npz provided by @caizhongang . , but I can not reproduce the SPIN training results, where the 3DPW PA-MPJPE reached 64.9.   The training log is 
[train.log](https://github.com/open-mmlab/mmhuman3d/files/9743112/train.log)

The  camera loss of first 400 step is very large:
2022-10-09 20:25:46,960 - mmhuman3d - INFO - Epoch [1][400/1563]	lr_backbone: 3.000e-05 lr_head: 3.000e-05, eta: 9:49:50, time: 2.324, data_time: 0.050, memory: 5855, keypoints3d_loss: 1.0694, keypoints2d_loss: 0.1184, vertex_loss: 0.1494, smpl_pose_loss: 0.0883, smpl_betas_loss: 0.0285, camera_loss: 58.8678, loss: 60.3219
2022-10-09 20:41:01,825 - mmhuman3d - INFO - Epoch [1][800/1563]	lr_backbone: 3.000e-05 lr_head: 3.000e-05, eta: 9:29:49, time: 2.287, data_time: 0.014, memory: 5855, keypoints3d_loss: 0.9419, keypoints2d_loss: 0.1422, vertex_loss: 0.1563, smpl_pose_loss: 0.0949, smpl_betas_loss: 0.0353, camera_loss: 0.6076, loss: 1.9782
2022-10-09 20:56:24,471 - mmhuman3d - INFO - Epoch [1][1200/1563]	lr_backbone: 3.000e-05 lr_head: 3.000e-05, eta: 9:14:33, time: 2.307, data_time: 0.014, memory: 5855, keypoints3d_loss: 0.8839, keypoints2d_loss: 0.1499, vertex_loss: 0.1574, smpl_pose_loss: 0.0991, smpl_betas_loss: 0.0363, camera_loss: 0.0001, loss: 1.3266
